data/tutorials/gs_00_up_and_running.md

```diff
@@ -108,6 +108,8 @@ On Unix, it's essential to initialise opam, which will (if needed) install the O
 
 This step is done automatically by the Platform Installer, [see below](#up-and-running-with-the-platform-installer).
 
+This step is done automatically by the Platform Installer, [see below](#up-and-running-with-the-platform-installer).
```

Comment:
Is this paragraph repeated above? It looks like it's in there twice, unless I'm misreading it.